docs/source/rest.html.md.erb

```diff
@@ -33,7 +33,15 @@ Replace `{objectID}` by the objectID you whish to query.
 $ curl -X GET 'https://places-dsn.algolia.net/1/places/111781_17807753'
 ```
 
-**Note:** The relative path prefix `/1/` indicates that we are currently using version 1 of the API.
```

Comment:
Why was this removed?

Reply:
Because it’s already stated a few lines above.

Reply:
The previous note applies to the search endpoint. Do we want to have all endpoints aligned on the search version at all times?